src/wp-includes/rest-api/endpoints/class-wp-rest-widgets-controller.php

```diff
@@ -204,6 +204,9 @@ public function create_item( $request ) {
 
 		$response->set_status( 201 );
 
+		// Triggers _delete_option_fresh_site to mark the site as no longer fresh.
+		do_action( 'wp_ajax_save-widget' );
+
```

Comment:
@kevin940726 The `wp_ajax_save-widget` event is specifically for triggering to save a widget via Ajax. Firing that event will call core’s `wp_ajax_save_widget()` function, which will resave the widget and invoke `wp_die`. It'll also call any other callbacks registered to it. 

I don’t think this is the behavior you’re wanting here as the widget has already been saved.

Reply:
Oh I didn't know it will re-call the function! I changed to go for the second approach described in the description now 👍